components/kyma-environment-broker/internal/process/input/builder.go

```diff
@@ -245,6 +247,12 @@ func (f *InputBuilderFactory) initProvisionRuntimeInput(provider HyperscalerInpu
 		},
 	}
 
+	// TODO: omit this section
+	//TODO Maper made a remark here!
```

Comment:
what can we do with this comment? :)

Reply:
My internal note. :) Removed in the next commit.